addons/web_tour/static/src/js/tour_service.js

```diff
@@ -32,7 +33,8 @@ return session.is_bound.then(function () {
     }
     return Promise.all(defs).then(function (results) {
         var consumed_tours = session.is_frontend ? results[0] : session.web_tours;
-        var tour_manager = new TourManager(rootWidget, consumed_tours);
+        const autoConsume = session.tour_disable || device.isMobile;
```

Comment:
The name `autoConsume` is confusing to me, in particular because it is a verb.
From what I can see, it simply means "disabled", or "consumed", or "done".